core/plugin.go

```diff
@@ -85,6 +85,16 @@ var (
 		metricPluginsStopping,
 		metricPluginsDead,
 	}
+
+	// Has to be index parallel to PluginState*
+	stateToDescription = []string{
```

Comment:
Hence the "might" - I wasn't quite sure if these tags are available for other types, too, but apparently they are not 🙄  In that case - plain old mapping it is.